src/hal/utils/scope_vert.c

```diff
@@ -836,29 +836,29 @@ static void chan_sel_button(GtkWidget * widget, gpointer gdata)
                     "the record length to allow for more channels"));
             gtk_dialog_run(GTK_DIALOG(dialog));
             gtk_widget_destroy(dialog);
-	    return;
-	}
-	if (chan->name == NULL) {
-	    /* need to assign a source */
-	    if (dialog_select_source(chan_num) != TRUE) {
-		/* user failed to assign a source */
-		/* force the button to pop back out */
-		ignore_click = 1;
-		gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(widget), FALSE);
-		return;
-	    }
-	}
-	vert->chan_enabled[chan_num - 1] = 1;
+            return;
+        }
+        if (chan->name == NULL) {
+        /* need to assign a source */
+            if (dialog_select_source(chan_num) != TRUE) {
+            /* user failed to assign a source */
+            /* force the button to pop back out */
+                ignore_click = 1;
+                gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(widget), FALSE);
+                return;
+            }
+        }
```

Comment:
I am only a bit unsure about how to indent comments like these best.
IMHO they belong over the if statement as they describe the whole body, but I wanted to keep changing only whitespaces.
But this way it looks also a bit wrong.

But indenting this 'correctly' would associate it with the next if statement:
```diff
         if (chan->name == NULL) {
-        /* need to assign a source */
+            /* need to assign a source */
             if (dialog_select_source(chan_num) != TRUE) {
```

Reply:
It's a subjective question of aesthetics, but my preference is that comments in the body of an if statement should be indented like the rest of the contents of the body, and should explain what the situation is now that we've entered into the body of the "if".

If a comment explains what the "if" is checking, then it should go above the "if" (and should start on the same column as the "if"), and probably end with a question mark for clarity.

And yes, i agree that makes the situation above somewhat ambiguous: does the comment explain the context inside the body, or does it explain the check that the next "if" is about to perform?  You just have to read it and figure it out!

Reply:
Okay it was the first one. I fixed that. But now is a new approval needed, sorry :D